api/config/settings.py

```diff
@@ -181,12 +246,49 @@
         'PASSWORD': os.getenv('DATABASE_PASSWORD'),
         'HOST': os.getenv('DATABASE_HOST'),
         'PORT': '',
-        }
+    }
 }
 DATABASES['default']['ATOMIC_REQUESTS'] = True
 DATABASES['default']['CONN_MAX_AGE'] = env.int(
     'DATABASE_CONN_MAX_AGE', default=0)
 
+# JWT settings
+
+JWT_AUTH = {
```

Comment:
Hello @IssaIan and @a-braham ,
JWT_AUTH dictionary has been redefined, it was defined here, it was initially defined in line 124.
You should also check on the travis build.

Reply:
Changed...Thanks for that